src/commands/webSocketTerminal.ts

```diff
@@ -624,7 +624,10 @@ function terminalConfigForUri(
   }
 
   return {
-    name: api.config.serverName && api.config.serverName != "" ? api.config.serverName : "iris",
```

Comment:
Not sure I like this part of the change. The host:port/prefix string can be very long and likely will be cut off in the UI by default.

Reply:
Fair point. I have reverted this.